DeleteEligibilityCheckActivity/handler.ts

```diff
@@ -63,7 +63,15 @@ export const getDeleteEligibilityCheckActivityHandler = (
               if (err.code === 404) {
                 return fromEither(right("NOT FOUND"));
               }
-              return fromEither(left(new Error(`QueryError: [${err}]`)));
+              // this condition address the case we want the activity to throw, so the orchestrator can retry
+              const queryError = new Error(
```

Comment:
can you please fix in the code above the double mapping / casting from fiscalCode to NonEmptyString?

Reply:
Do you mean this 87fb0ed?

Reply:
yes :)